torch/fx/node.py

```diff
@@ -78,6 +78,9 @@ def _type_repr(obj):
     return repr(obj)
 
 def _get_qualified_name(func: Callable[..., Any]) -> str:
+    # higher_order_operators
+    if isinstance(func, torch._ops.HigherOrderOperator) and func.name() in torch._ops._higher_order_ops:
+        return f"torch.ops.higher_order.{func.name()}"
```

Comment:
Does this also change `map`? If so, that may be a problem, because map is still torch.ops.map.

The check isn't too robust:
1) Ideally cond's `__name__` is "cond" and `__module__` is "torch.ops.higher_order". (Is this the case?)
2) If so, then we should return `f"{func.__module__}.{func.__name__}`

Reply:
This pr doesn't deal with map. My plan is to experiment on cond a bit to see how it goes then do it for map in the next pr.

1. cond's \_\_name\_\_ is "cond" but \_\_module\_\_ is "torch._ops". I can check a bit how to make its \_\_module\_\_ torch.ops.higher_order.
2. Yeah, I agree.